pybrake/test_queues.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+from .queues import QueueMetric, QueueStats, _QueueStat
+
+
+def test_queue_metric_start_end():
+    metric = QueueMetric(queue="test")
+    metric.end()
+    assert metric.end_time is not None
+
+
+def test_queue_metric_key():
+    metric = QueueMetric(queue="test")
+    assert ("test", metric.start_time // 60 * 60) == metric._key()
+
+
+def test_queue_stats_performance_stats():
+    metric = QueueMetric(queue="foo_queue")
+    metric._groups = {'redis': 24.0, 'sql': 0.4}
+    stats = QueueStats(**{"config": {"performance_stats": False}})
+    metric.end()
+    assert stats.notify(metric) is None
+
+
+def test_queue_stats_queue_stat_value():
+    metric = QueueMetric(queue="foo_queue")
+    stat = _QueueStat(queue=metric.queue, time=metric.start_time)
+    assert stat.__dict__.get('queue') == 'foo_queue'
+
+
+def test_queue_stats_queue_stats():
+    metric = QueueMetric(queue="foo_queue")
+    stats = QueueStats(**{"config": {
+        "performance_stats": True,
+        "queue_stats": False,
+        "error_host": "https://api.airbrake.io",
+        "apm_host": "https://api.airbrake.io",
```

Comment:
Do we _really_ need to pass `error_host` and `apm_host` in these test cases where we don't check for their value?